packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -141,15 +149,15 @@ export default function OrgNav({
         />
 
         <ul className={ulContainer}>
-          {paymentVariant && current?.paymentStatus && (
+          {paymentVariant && showPaymentStatus && (
```

Comment:
Is there a reason why we don't add the `paymentVariant` check to `showPaymentStatus`?

Reply:
Good call 👍